README.md

```diff
@@ -202,6 +202,77 @@ If query text is present, the translation of query text based on input language
 
 If the query text is absent and audio url is present, then the audio url is downloaded and converted into text based on the input language. Once speech to text conversion in input language is finished, the same process mentioned above happens. One difference is that by default, the paraphrased answer is converted to voice irrespective of the output_format since the input format is voice.
 
+### `POST /v1/chat`
+
+#### API Function
+API is used to generate activity/story based on user query and translation of text/audio from one language to another language in text/audio format. To achieve the same, Bhashini has been integrated. OCI object storage has been used to store translated audio files when audio is chosen as target output format.
+
+```commandline
+curl -X 'POST' \
+  'http://127.0.0.1:8000/v1/chat' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "input": {
+    "language": "en",
+    "text": "string",
+    "audio": "string",
+    "context": "parent"
+  },
+  "output": {
+    "format": "text"
+  }
+}'
+```
+
+#### Request
+| Request Input       | Value                                                     |
+|:--------------------|-----------------------------------------------------------|
+| `input.language`    | en,bn,gu,hi,kn,ml,mr,or,pa,ta,te                          |
+| `input.text`        | User entered question (any of the above language)         |
+| `input.audio`       | Public file URL Or Base64 encoded audio                   |
+| `input.context`     | parent, teacher (default value is parent, if not passing) |
+| `output.format`     | text or audio                                             |
+
+Required inputs are `text`, `audio` and `language`.
+
+Either of the `text`(string) or `audio`(string) should be present. If both the values are given, `text` is taken for consideration. Another requirement is that the `language` should be same as the one given in text and audio (i.e, if you pass English as `language`, then your `text/audio` should contain queries in English language). The audio should either contain a publicly downloadable url of mp3 file or base64 encoded text of the mp3.
+If output format is given as `text` than response will return `text` format only. If output format is given as `audio` than response will return `text` and `audio` both.
+
+```json
+{
+   "input": {
+      "text": "How to Teach Kids to Play Games", 
+      "language": "en"
+   },
+   "output": {
+      "format": "text"
+   }
+}
+```
+
+#### Successful Response
+
+```json
+{
+   "output": {
+      "text": "string",
+      "audio": "string",
+      "language": "en",
+      "format": "text|audio"
+   }
+}
+```
+
+#### What happens during the API call?
```

Comment:
Good to have 
- a sequence diagram that explains this flow for /chat
- a sequence diagram that explains this flow for /query

This will be a good information for the adopters.

Reply:
Will add it as part of the microsite.